routers/web/repo/view.go

```diff
@@ -1122,25 +1123,42 @@ func RenderUserCards(ctx *context.Context, total int, getter func(opts db.ListOp
 	ctx.HTML(http.StatusOK, tpl)
 }
 
+func renderUserList(ctx *context.Context, pageType string, total int, getter func(opts db.ListOptions) ([]*user_model.User, error), tpl base.TplName) {
+	ctx.Data["Title"] = ctx.Tr(pageType)
+	ctx.Data["CardsTitle"] = ctx.Tr(pageType)
+	RenderUserCards(ctx, total, getter, tpl)
```

Comment:
It becomes a Russian matryoshka(nesting) doll now.

Why make it `Watchers/Stars -> renderUserList -> RenderUserCards` ?

`RenderUserCards` was only used by `Watchers/Stars`, so what's benefit to introduce the new `renderUserList` patch?

Reply:
I do not like duplicate either, but I don't understand why not just keep one function (say, merge renderUserList and RenderUserCards)?

Reply:
I merged the functions and simplified the calls to `RenderUserCards` by aligning the function signatures of `GetStargazers`. Now we avoid creating middleman functions